bin/packages/build-types.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable eslint-comments/disable-enable-pair, no-console */
+
+/**
+ * External dependencies
+ */
+const execa = require( 'execa' );
+const glob = require( 'fast-glob' );
+const path = require( 'path' );
+
+const tscPath = path.resolve(
+	__dirname,
+	'..',
+	'..',
+	'node_modules',
+	'.bin',
+	'tsc'
+);
```

Comment:
I recall some potential issues when trying to reference `node_modules` directly:

https://github.com/WordPress/gutenberg/pull/20215#discussion_r379176166

Not sure if it's something we expect to be problematic here, also given that this is referencing the top-level `node_modules/.bin` and not a package directory.

Reply:
Resolving the package bin can be an involved process, I'd expect this to be reliable as long as we're on npm and the way packages are installed doesn't change radically. The failure should be obvious if and when it arises.